LogicalQ/Experiments.py

```diff
@@ -252,6 +254,19 @@ def _basic_experiment_core(task_id, circuit, noise_model, backend, method, shots
 
     return task_id, result
 
+def parity_experiment_core(task_id, circuit, noise_model, backend, method, shots, stabilizer_code):
```

Comment:
Is this _experiment_core function there for the sake of experiments that run both physical and logical realizations of the same circuit?